DOCS-462: trace_log system table and trace_log configuration section docs. (#7842)
ClickHouse/ClickHouse: docs/en/operations/system_tables.md

You can specify an arbitrary partitioning key for the `system.query_log` table in the [query_log](server_settings/settings.md#server_settings-query-log) server setting (see the `partition_by` parameter).

## system.trace_log {#system_tables-trace_log}

Contains stack traces collected by the sampling query profiler.

ClickHouse creates this table when the [trace_log](server_settings/settings.md#server_settings-trace_log) server configuration section is set. Also the `query_profiler_real_time_period_ns` and `query_profiler_cpu_time_period_ns` settings should be set.

To analyze logs, use the `addressToLine`, `addressToSymbol` and `demangle` introspection functions.

Columns:

- `event_date`([Date](../data_types/date.md)) — Date of sampling moment.
- `event_time`([DateTime](../data_types/datetime.md)) — Timestamp of sampling moment.

> - `revision`([UInt32](../data_types/int_uint.md)) — ClickHouse server build revision.

Reviewer: needs a comment that it doesn't directly relate to human-readable release versions

- `timer_type`([Enum8](../data_types/enum.md)) — Timer type:

    - `Real` represents wall-clock time.
    - `CPU` represents CPU time.

- `thread_number`([UInt32](../data_types/int_uint.md)) — Thread identifier.

> - `query_id`([String](../data_types/string.md)) — Query identifier. Information about a query you can get from the [query_log](#system_tables-query_log) table.

Reviewer: ```suggestion
- `query_id`([String](../data_types/string.md)) — Query identifier that can be used to get details about a query that was running from the [query_log](#system_tables-query_log) system table.
```

> - `trace`([Array(UInt64)](../data_types/array.md)) — Stack trace at the moment of sampling.

Reviewer: ```suggestion
- `trace`([Array(UInt64)](../data_types/array.md)) — Stack trace at the moment of sampling. Each element is a virtual memory address inside ClickHouse server process.
```

> - `trace`([Array(UInt64)](../data_types/array.md)) — Stack trace at the moment of sampling.

Reviewer: ```suggestion
- `trace`([Array(UInt64)](../data_types/array.md)) — Stack trace at the moment of sampling. Each element is a virtual memory address inside ClickHouse server process.
```
